packages/whosmysanta-backend/src/data/friend/index.js

```diff
@@ -1,32 +1,32 @@
 import mongoose from 'mongoose';
 import shortid from 'shortid';
 import FriendModel from './model';
-import {throwError} from '../../helpers';
 
 export const getFriends = ({first = 10, id}) =>
-  FriendModel.find(id && {id}).limit(first).exec().catch(throwError);
+  FriendModel
+    // Only pass id if defined
+    .find(id && {id})
+    .limit(first)
+    .exec()
+    .catch((err) => { throw new Error(err); });
 
 export const getFriendById = id =>
   FriendModel.findOne({_id: mongoose.Types.ObjectId(id)})
     .exec()
-    .catch(throwError);
+    .catch((err) => { throw new Error(err); });
```

Comment:
I think if we have a second line or a special logging thing, then let's do it. For now let's keep it simple, repetitive and less abstracted.